requirements/dev.txt

```diff
@@ -10,7 +10,7 @@ astroid==2.11.2
     # via
     #   pylint
     #   pylint-celery
-bleach==4.1.0
+bleach==5.0.0
```

Comment:
Dropped support for `Python 3.6`. See the [CHANGELOG](https://github.com/mozilla/bleach/blob/main/CHANGES) for details.